tests/components/samsungtv/test_media_player.py

```diff
@@ -264,9 +269,9 @@ def mock_now():
 
 async def test_send_key_autodetect_websocket_exception(hass, caplog):
     """Test for send key with autodetection of protocol."""
-    caplog.set_level(logging.DEBUG)
```

Comment:
Don't remove this.

Reply:
I don't remember removing this, but of course ! (that's why reviewers are here)